apps/filebrowser/src/filebrowser/templates/fb_components.mako

```diff
@@ -58,9 +59,15 @@ else:
         %endif
         <li>
           <ul id="editBreadcrumb" class="hue-breadcrumbs editable-breadcrumbs" data-bind="foreach: breadcrumbs" style="padding-right:40px; padding-top: 12px" title="${_('Edit path')}">
-            <li data-bind="visible: label.slice(-1) == '/'">
-              <a data-bind="click: show, attr: {'href': '${url('filebrowser:filebrowser.views.view', path=urlencode(''))}' + url}"><span class="divider" data-bind="text: label"></span></a>
-            </li>
+            %if RAZ.IS_ENABLED.get():
```

Comment:
window.RAZ_IS_ENABLED usable?

Reply:
then need to use inside the <li>? like below
so which one is good?
![Screenshot 2021-08-18 at 12 07 12 PM](https://user-images.githubusercontent.com/36241930/129849738-eb203be8-6ac7-4bb5-ad03-8c6b9d50afc5.png)